Sources/MapboxNavigation/Transitioning.swift

```diff
@@ -59,13 +59,29 @@ extension PresentAnimator: UIViewControllerAnimatedTransitioning {
             height = draggable.draggableHeight
         }
         
-        let finalFrame = CGRect(origin: CGPoint(x: 0, y: fromVC.view.bounds.height - height),
-                                size: CGSize(width: fromVC.view.bounds.width, height: height))
+        // In case of certain presentation styles on iPad `FeedbackViewController` is placed incorrectly.
+        // This check adjusts its size in such case.
+        var y = fromVC.view.bounds.height - height
+        var width = fromVC.view.bounds.width
+        let modalPresentationStyles: [UIModalPresentationStyle] = [.pageSheet, .formSheet]
+        if UIDevice.current.userInterfaceIdiom == .pad,
+           modalPresentationStyles.contains(fromVC.modalPresentationStyle) {
+            y = UIScreen.main.bounds.size.height - height
```

Comment:
Just curious, can't we use the same calculation on every platform? Will it break some use case which isn't obvious just by looking at this code?

Reply:
Yep, made some changes to make this fix more generic. Also fixed some minor animation transitioning issue.